cypress/support/elements/common/PrimaryWorkspace.js

```diff
@@ -84,9 +68,6 @@ class RightNav{
       return cy.get('.list.'+section+' [data-test='+section+'-list-items] .footer');
     }
 
-    starCanvasItem(tab,section,title){
-        cy.getCanvasItemTitle(section).contains(title).parent().parent().siblings('.icon-holder').find('.icon-star').click();
-    }
     getCanvasStarIcon(tab,section,title){
         return this.getCanvasItemTitle(tab, section).contains(title).parent().parent().siblings('.icon-holder').find('.icon-star');
     }
```

Comment:
This function should also removed from here since it should be in ResourcePanel

Reply:
done